frontend/src/data/useData.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react'
+import { Room } from '../types/room';
+
+interface Data {
+  rooms: Room[];
+}
+
```

Comment:
There are unnecessary line breaks in the code. Remove the extra line breaks for better readability.